src/ansys/templates/python/common/{{cookiecutter.__project_name_slug}}/doc/source/_templates/sidebar-nav-bs.html

```diff
@@ -1,12 +1,11 @@
+{% raw %}
```

Comment:
i think we can remove this file, and add manually upon request on individual lib.

Reply:
I agree, thanks for suggesting this, @Revathyvenugopal162.